### PR TITLE
Topic/qiy scheme releases

### PR DESCRIPTION
**Pull request by 14665**

### CODE REVIEW

**Review by fdriesenaar — APPROVED** [review]

File: Qiy Scheme Releases.md [hunk 1]
```diff
@@ -3,22 +3,27 @@
...
 The Qiy Scheme defines a framework for individual users, companies and governmental organisations to safely control and exchange personal data. This document describes what the requirements are for new releases of the Qiy Scheme and how these new releases are managed.
 
-# Content
-
 # 1 Qiy Scheme: new releases
-This document is a policy document which describes what the requirements are for new releases of the Qiy Scheme and how these new releases are managed.
+This document is a policy document which describes the requirements for new releases of the Qiy Scheme and how these new releases are managed.
+
+# 2 Versioning
+The Supervisory Authority of the Qiy Foundation decides upon the release of new versions of the Qiy Scheme:
```
Comment (mlouman) [inline Qiy Scheme Releases.md]: The Executive Board of the Qiy Foundation decides upon the release of new versions of the Qiy Scheme:

File: Qiy Scheme Releases.md [hunk 2]
```diff
@@ -3,22 +3,27 @@
...
+* The Supervisory Authority will release a major version when it introduces incompatible changes.
```
Comment (mlouman) [inline Qiy Scheme Releases.md]: Supervisory Authority = Executive Board

File: Qiy Scheme Releases.md [hunk 3]
```diff
@@ -3,22 +3,27 @@
...
+* The Supervisory Authority will release a minor version when functionality is added in a backwards-compatible manner.
```
Comment (mlouman) [inline Qiy Scheme Releases.md]: Supervisory Authority = Executive Board

File: Qiy Scheme Releases.md [hunk 4]
```diff
@@ -3,22 +3,27 @@
...
+* The Supervisory Authority will release a patch version for backwards-compatible (bug) fixes.
```
Comment (mlouman) [inline Qiy Scheme Releases.md]: Supervisory Authority = Executive Board

File: Qiy Scheme Releases.md [hunk 5]
```diff
@@ -3,22 +3,27 @@
...
 
-# 2 
-The Supervisory Authority of the Qiy Foundation decides upon the release of updates and new version of the different elements (governance, legal and technical) of the Qiy Scheme.
+# 3 Rules
+New versions of the Qiy Scheme will apply from the date of release.
+The Supervisory Authority will follow these rules:
```
Comment (mlouman) [inline Qiy Scheme Releases.md]: The Supervisory Authority = The Executive Board

**Review by mlouman — CHANGES_REQUESTED** [review]

**Pull request merged**